src/rhsmlib/dbus/objects/register.py

```diff
@@ -366,12 +377,18 @@ def RegisterWithActivationKeys(self, org, activation_keys, options, connection_o
             Locale.set(locale)
             cp = self.build_uep(connection_options)
 
+            # Remove 'enable_content' option, because it will not be proceed in register service
+            enable_content = self._remove_enable_content_option(options)
+            if enable_content is not None:
+                log.debug('Ignoring "enable_content" option, because it cannot be used with activation keys')
```

Comment:
if `enable_content` cannot be used with `RegisterWithActivationKeys()`, shouldn't it be better rejected outright, rather than silently dropping it? this way there is no risk that is accidentally passed it and "silently works"

Reply:
Actually the `enable_content` should be option only for `Register()` method. Thus this block of code should not be here and the exception will be raised automatically.